index.html

```diff
@@ -3,7 +3,7 @@
 
 <head>
     <meta charset="UTF-8">
-    <title>Gaucho Task Launcher</title>
+    <title>Gaucho Task Runner</title>
```

Comment:
Why this change in the title?

The title was change on purpose to Launcher, as I think it defines better what gaucho is for.

If you think another title/name is better, feel free to open an issue so it can be deiscussed

Reply:
I don't remember have made this change (?) maybe I just copy it from the main branch or something XD, sorry.

Reply:
No Problem, thats what Code reviews are for :+1: